compose/foundation/foundation/src/uikitMain/kotlin/androidx/compose/foundation/gestures/UikitScrollable.uikit.kt

```diff
@@ -19,8 +19,15 @@
 package androidx.compose.foundation.gestures
 
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.input.pointer.PointerEvent
+import androidx.compose.ui.unit.Density
+import androidx.compose.ui.unit.IntSize
 
 @Composable
-internal actual fun platformScrollConfig(): ScrollConfig =
-    TODO("implement native platformScrollConfig")
+internal actual fun platformScrollConfig(): ScrollConfig = UiKitConfig
+
+private object UiKitConfig : ScrollConfig {
+    override fun Density.calculateMouseWheelScroll(event: PointerEvent, bounds: IntSize): Offset =
+        error("No scrolling events expected, received: $event")
```

Comment:
```suggestion
        // 64 dp value is taken from ViewConfiguration.java, replace with better solution
        return event.changes.fastFold(Offset.Zero) { acc, c -> acc + c.scrollDelta } * -64.dp.toPx()
```
Like in Android. Maybe can be useful in the future.

Reply:
Done